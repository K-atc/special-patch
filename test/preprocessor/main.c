#include <stdio.h>
#include "calc.h"

int main() {
    int a = 1;
    int b = inc(1);
    printf("a + b = %d\n", (a + b));
}