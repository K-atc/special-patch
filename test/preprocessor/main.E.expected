# 0 "main.c"
# 0 "<built-in>"
# 0 "<command-line>"
# 0 "<command-line>" 2
# 1 "main.c"
#include <stdio.h>
# 1 "main.c"
# 2 "main.c" 2
# 2 "main.c"
# 1 "calc.h" 1



# 3 "calc.h"
static inline int inc(int a) {
    return a + 1;
}
# 3 "main.c" 2

int main() {
    int a = 1;
    int b = inc(1);
    printf("a + b = %d\n", (a + b));
}
